tests/test_aging.c

```diff
@@ -0,0 +1,532 @@
+/**
+ * @file test_aging.c
+ * @author Ondrej Kusnirik <Ondrej.Kusnirik@cesnet.cz>
+ * @brief tests for aging utility
+ *
+ * @copyright
+ * Copyright (c) 2018 - 2022 Deutsche Telekom AG.
+ * Copyright (c) 2018 - 2022 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <string.h>
+#include <dirent.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <signal.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <setjmp.h>
+#include <stdarg.h>
+#include <cmocka.h>
+
+#include <libyang/libyang.h>
+
+#include "sysrepo.h"
+#include "sysrepo_types.h"
+#include "test_common.h"
+#include "config.h"
+
+#define TIME_MAX 100000
+#define NUM_OF_FILES 30
+
+typedef struct {
+    sr_conn_ctx_t *conn;
+    sr_session_ctx_t *sess;
+    char *n_path;
+    char *a_path;
+    char *path_to_file;
+    char *pidfile;
+    uint32_t len_npath;
+    uint32_t len_apath;
+    pid_t pid;
+} test_data_t;
+
+/* Allocs memory, memory needs to be freed */
```

Comment:
Redundant, understood from the return value type.

Reply:
Removed